src/numbuf/python/src/pynumbuf/memory.h

```diff
@@ -44,7 +50,11 @@ class FixedBufferStream : public arrow::io::OutputStream,
     DCHECK(position_ + nbytes <= size_) << "position: " << position_
                                         << " nbytes: " << nbytes << "size: " << size_;
     uint8_t* dst = data_ + position_;
-    memcpy(dst, data, nbytes);
+    if (nbytes >= (1<<20)) {
```

Comment:
will do, where's a good place to put it? It's worth defining things like `#define MB (1<<20)` and using that for better readability.